site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,409 @@
+
+In the past, this rasterizer cache was tied with OpenGL and used OpenGL methods directly. This PR introduces a class called `TextureRuntime`. This is used to abstract away texture operations from the rasterizer cache. By doing this, it allows for other graphic APIs to be easily ported over to Citra, which we will get into more later in this report.
+
+## 3GX Plugin Loader ([#6172](https://github.com/citra-emu/citra/pull/6172)) by [PabloMK7](https://github.com/PabloMK7)
+
+`.3gx` plugins are executable files that are mapped into memory and executed when the game process runs. When the game runs, the `.3gx` plugins also run 
+simultaneously in a new thread, which grants it the same access rights as the game does. This allows developers to expand the behavior of games in a way that would normally be impossible, such as with an Action Replay code.
+
+Using 3GX plugins is another way to add that extra customizability to your 3DS games in a way that the unmodded 3DS can’t do. Many projects use 3GX plugins, such as the Mario Kart 7 modpack [CTGP-7](https://ctgp-7.github.io/). 
+In the past, due to Citra not supporting 3GX plugins, a “lite” version of this modpack was offered to Citra players. But this came with nowhere near as many tracks and features as the main modpack! The addition of this plugin loader grants Citra users with more modpacks to use to alter your game in more ways than before!
+
+Plugins are loaded from the emulated 3DS SD card. Citra follows the same path as the Luma3DS 3GX Plugin Loader to apply 3GX plugins. Instructions for launching `.3gx` plugins can be found on our [Game Modding](https://citra-emu.org/help/feature/game-modding/#:~:text=Using%203GX%20plugins) page.
+
+## Add Nearest Neighbour Texture Filter ([#6189](https://github.com/citra-emu/citra/pull/6189)) by [venkatrao1](https://github.com/venkatrao1)
+
+If you hadn’t seen in our previous Progress Report, [Texture Filters](https://citra-emu.org/entry/citra-progress-report-2020-q2/#new-features:~:text=Texture%20Filters) were added to Citra!
+With texture upscaling, Citra will use one of several algorithms to automatically upsize the image to a higher resolution. 
+
+Nearest Neighbour is a texture filter which is particularly effective at making text stand out and look bolder! With the addition of another highly requested texture filter, you can have more control over how your gameplay looks than ever before!
+
+All of these texture filters, including our shiny new Nearest Neighbour texture filter, can be found in `Emulation -> Configure -> Graphics -> Renderer` in Citra! (`Citra -> Preferences` on MacOS)
+
+{{< juxtapose "cc0544e8-aba2-11ed-b5bd-6595d9b17862" >}}
+
```

Comment:
(For me, there is a huge gap in the article preview here. Not sure if that is an issue on my side)

Reply:
I have no idea how to fix that, I think it's something to do with how the juxtapose works but I didn't implement functionality for that.